sihl/src/web_migration.ml

```diff
@@ -0,0 +1,63 @@
+let log_src = Logs.Src.create "sihl.middleware.migration"
+
+module Logs = (val Logs.src_log log_src : Logs.LOG)
+
+exception Exception
+
+let template migrations =
+  let migrations = migrations |> List.map fst |> String.concat ", " in
+  Format.asprintf
+    {|
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Pending migrations</title>
+  <style>
+    %s
+  </style>
+</head>
+
+<body class="antialiased">
+  <div class="py-4 sm:py-12">
+    <div class="max-w-8xl mx-auto px-4 sm:px-6 lg:px-8">
```

Comment:
`max-w-8xl` does not seem to be a valid class, please remove

Reply:
I removed the custom template and just throw an exception.

Reply:
Haha that works :D